libs/nft-contract/prodtest.tokens.json

```diff
@@ -0,0 +1,28 @@
+{
+  "0": "141JDhjWvgrqvCvmcG2HT3HTk6FnBjm8LYan6rMtthLWjJCZuUNAKnqaeG8Bovor8Q55sBgF8X3n6nuJtTxKjEgTL",
+  "1": "NgdDLrbbbCVtzBWB36JHJ4Jvryk118wmiqUSoJsR8crVu4uPjNNpMbNygfpQ6xspJQjeMZwCJL9Z2qzEwRro8wZD4",
+  "2": "MGFC2cvvLmvRUx7LazesLtnDsB8ihwvnexJF3mdWicW6QpscfC4ySMmyHd11CNuvmgccND2ddL9SiyTjB9W5nKVWz",
+  "3": "CkgYyfrkm7dhtac26UAkvSb9tEinfkk81sTGZEcmMaRSJUtsfFv6wyUXRzKb3L3QGwa8QXb6c1hLED7hUoyzvLi7M",
+  "4": "CB8iuNwDeZdFXZDTwXeNPH6MHm6KHXdUzcdx9yBHJPbbfdpy9mtiaRXrGdqVmKqEnoZTaNsFkyG7NtnDXrUZxZ9B9",
+  "5": "FowQ92YKQ95JxGaiu8JXkVpyQoPYeU5cUVSTS2X6V9Lr8bJ6zfcwu8SqFRa55D6RBda4vmUQRNB84a5yrR4PYTudc",
+  "10": "yi4wRRbVX18kGkwphAM3WgMxPgFJT3kzuX6qkuGiEsTqavqaBejkPriFQwaPgEpCXT4SYfQiobwpfSwMDFbNxLkB",
+  "11": "13rmygVLPfJBKfQhhKF8PyPdyaeh43Soq6iPo1N4j2RE1wX59UbTSJANEpY4bHBNe3emGC5FSBCesXuuDkrWC4yPx",
+  "12": "BE2saqKExGDn9nk4wWCPTRCMA9RGMVUgdmPhebUJYwSw2S9b18Yyfu2eTj55CPYTaUXcPPm9KrbVexVBugnqWZKY",
+  "13": "2tTUc1KCw6VmDVzLJmJRWYX51Pnt3zqKmHW1hi4x3LUKUpYtf7FyUnqC3Wsm8fdJ33NiSoQnemxedxBUcuaMrPtS6",
+  "14": "JNhxsbhwthLHtwktTRGDpC4xdy1TBMxiMhs2kdHF3zAsaJbJPmxfbS7LW2wJD9PDB7RQbk7eNGtzcVA8dc8nkpPtF",
+  "100": "DKP6q1fSL8JRGqiZop1CG6Ew7trttFX7JSBxASncnubBXbwZiXkPYiqpVZzYA2uYPBBeMZm1zgtbMWY1XVcfVZ4wt",
+  "101": "FaHxRz2tTkJWyiLFCmhVwPNmoHp51Rk7skU9zQAjtmg9w5p3uyjoc7m5ZvgSqD18jtX5hPKdZ2UshbfGQYdQMHqSJ",
+  "102": "AUL9fe1ZVTDytRg9n2i7t8Au2TacvLs4NRn7e4GniY6LW6y1fUz6iF4QrUTP2bqUiswYCUwKchJJjFT6FvJVGCmXx",
+  "103": "DLqnhheJG7mu1Fn2jL7fUuuhG5pLSqB4B7TzPKcRL9nfncebRjCiTGkSXrXLav4JnydivMhDhZutZHhADct9NshvS",
+  "104": "9K7tWmnQ6F8p11X4Dff9e65fPf1cEJbYQzLZcbtZgSr2KTCNxkoPbPS9Q2QVrXymyTPruC69PEUb1aEcCGJLYEeFc",
+  "1000": "JomPbgbnx35dTuv7QNf9EQZ7nKDkUA7hfKqmaGFijP7rQUzn8w1FNRt628ftZgsePJaaLokkLeayfykfmoc5APFke",
+  "1001": "76QTsDfMkNYE7pNqVgrrjBxyMDhJdZxJ7kNxtiP6JkL2V4J4V6cgNorguhMUpaeYrfiSyMaYGVV1jnHyfTufpLso1",
+  "1002": "LUL4ejQLJQ9svpNUzgz1E1iQVZujEiPecxLdG8Mr6QbEYApMAT7J3vz2SHqd6Ck7nLWMtXLzXXvzK1TMkd2yZFT9t",
+  "1003": "54aXfhYi2nXCYjFFa8KF4urv9qKN3edskgwfc87TMePXJbwj7MRsB1jCPhLREUPdvXhdpntyTy4jasmAbigEsrrca",
+  "1004": "AamPQNM4Sd27hLHKZajk9MtkUUuiZbWHmAWHLyq5ZeToKns1S5Ut7ceRgtFmfAGL3yMEC68pKm4Vf1rBQPxKkvBfY",
+  "10000": "KwqCyyGbErBGD1j6kgdbvMbzsGmhhNy6Hatv9x1TTx4esbxwZi5vEf35ZLcXoGBfE2SuUQc6rYmgYKBMFXA9zp4su",
+  "10001": "7zxnNJrNEB4MACsX2EzUXhsebhY8wVTVSRSQh1ZV1ACFKxx6ePHT4wr1Lv3VRfcnhgbtUzD7W4nGHzfRfN3XRGWR1",
+  "10002": "HK3Gg1kvkTmTqNYZsw6pEMa2T7jLyezEXW233BV3FJTiWmJpiaVkh1mkwQBLm2d4HSJA42ABj2jLx9AXsJPQcQ75u",
+  "10003": "5qrt7ftD93Y68SZ1QvZ5bYcMKgboAJyxg4sRwN2wM6SaBX2VnSn1ukRABkUikYw9vodNhF2xKefUVhbK3qXtQVBVm",
+  "10004": "83rsyGBAnCqvWE2R8tMx1ZwU1NGgXZu13NJ3w2xMbNayo5vLhzi7pfuK19L82aZuWW74k248prAv5whSjafrHGVdD"
```

Comment:
Comitted because we want to persist this info.

Although - maybe I didn't need to: it's determinstic (ie, can regenerate at whim).  I might remove this file in a near-future PR